frontend/src/components/pages/rp-connect/tasks.tsx

```diff
@@ -1,5 +1,5 @@
 export const MIN_TASKS = 1,
-  MAX_TASKS = 90;
+  MAX_TASKS = 72;
```

Comment:
Where does this value come from?

Reply:
Initially I set a high value since we did not know what would be the max. Now we settled at 72 as max in cloud.